### DISCUSSION TITLE
I’m a newbie, I got a permission denied error

### DISCUSSION THREAD

**Discussion by frenemj8** [GyulyVGC/sniffnet#55] [Q&A] [answered]
Morning you all ,
macOS 10.15.7 installed without hiccups,
program running but when selecting any interfaces ..i get 
"libpcap error : ( cannot open BPF device )  /dev/bpf0 : permission denied

Would a fine helping spirit point me in the right direction ...please ?

Regards !

**Comment by GyulyVGC** [comment, marked as answer]
> Morning you all , macOS 10.15.7 installed without hiccups, program running but when selecting any interfaces ..i get "libpcap error : ( cannot open BPF device ) /dev/bpf0 : permission denied
> 
> Would a fine helping spirit point me in the right direction ...please ?
> 
> Regards !

Hi @frenemj8.
You just have to launch the application as administrator:

`sudo sniffnet`

**Comment by frenemj8** [comment]
damm , i didn’t realized.

thanks and have a great day.

Michel

Le 6 févr. 2023 à 11:52, Giuliano Bellini ***@***.***> a écrit :

﻿

Morning you all , macOS 10.15.7 installed without hiccups, program running but when selecting any interfaces ..i get "libpcap error : ( cannot open BPF device ) /dev/bpf0 : permission denied

Would a fine helping spirit point me in the right direction ...please ?

Regards !

Hi @frenemj8<https://github.com/frenemj8>.
You just have to launch the application as administrator:

sudo sniffnet

**Reply by GyulyVGC** [reply]
I'm happy it fixed your problem. Please mark my message as answer so that other people can view this discussion as answered.

Have a good day!

**Reply by frenemj8** [reply]
grazie molto
